src/jama/create/create.py

```diff
@@ -143,8 +153,15 @@ def from_text(base_url, content_dict):
         
 
     url = base_url + '/rest/latest/items/'
-    jama_resp = requests.post(
-        url, json=to_post_obj, auth=(os.environ['JAMA_USER'], os.environ['JAMA_PASS']))
+    if "DB_HOST" not in os.environ:
```

Comment:
Similar to line 53.
You can simply use the `api_caller.post()` in `Jama/api_caller` to do the same thing.
I think using the same caller function can make our code more maintainable.

Reply:
@yejingyu done